Lib/test/support/__init__.py

```diff
@@ -2756,6 +2760,60 @@ def fd_count():
 
     return count
 
+def run_pty(script, input=b"dummy input\r", env=None):
```

Comment:
I propose to add a docstring to document the side effect of import_module('pty'):

"""
Run a script in a temporary pseudoterminal created just for the script.

Raise SkipTest if the pty module cannot be imported.
"""